tools/perf/deterministic_run.py

```
@@ -0,0 +1,55 @@
+# Copyright (c) PyPTO Contributors.
```

Ruff (0.16.1)

[warning] The file is executable but no shebang is present

(EXE002)